analyzers/src/SonarAnalyzer.CFG/SonarAnalyzer.CFG.csproj

```diff
@@ -13,10 +13,14 @@
       <!-- This package is a dependency of Microsoft.CodeAnalysis.CSharp.Workspaces. It is safe to use since it's compatible with .Net Portable runtime -->
       <NoWarn>NU1701</NoWarn>
     </PackageReference>
+    <PackageReference Include="System.Collections" Version="4.3.0" />
     <PackageReference Include="System.Collections.Immutable" Version="1.1.37">
       <!-- Downgrade System.Collections.Immutable to support VS2015 Update 3 -->
       <NoWarn>NU1605, NU1701</NoWarn>
     </PackageReference>
+    <PackageReference Include="System.IO.FileSystem" Version="4.3.0" />
+    <PackageReference Include="System.IO.FileSystem.Primitives" Version="4.3.0" />
```

Comment:
How do these break the build? On CI only? I can still build it locally if I comment them.

Reply:
I've noticed now that only `System.IO.FileSystem` is needed and only on the `SonarAnalyzer.Common`. 
If missing from there they fail the local build. This is the message I get :
```
SonarAnalyzer.RuleDescriptorGenerator.csproj: [NU1605] Detected package downgrade: System.Text.Encoding from 4.3.0 to 4.0.11. Reference the package directly from the project to select a different version. 
 SonarAnalyzer.RuleDescriptorGenerator -> SonarAnalyzer -> Microsoft.CodeAnalysis.Workspaces.Common 1.3.2 -> Microsoft.CodeAnalysis.Common 1.3.2 -> System.Console 4.0.0 -> runtime.win.System.Console 4.3.1 -> System.Text.Encoding (>= 4.3.0) 
 SonarAnalyzer.RuleDescriptorGenerator -> SonarAnalyzer -> Microsoft.CodeAnalysis.Workspaces.Common 1.3.2 -> Microsoft.CodeAnalysis.Common 1.3.2 -> System.Text.Encoding (>= 4.0.11)
```